source/features/profile-hotkey.tsx

```diff
@@ -3,13 +3,11 @@ import select from 'select-dom';
 import features from '../libs/features';
 import {getUsername} from '../libs/utils';
 
-function init(): false | void {
+function init(): void {
 	const menuItem = select(`a[href="/${getUsername()}"]`);
 
 	if (menuItem) {
 		menuItem.dataset.hotkey = 'g m';
-	} else {
-		return false;
```

Comment:
Did you mean to use `?.`?

```js
select(`a[href="/${getUsername()}"]`)?.dataset.hotkey = 'g m';
```

Reply:
You cant since your assigning something

Reply:
Indeed, not valid at all: 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining#Optional_chaining_not_valid_on_the_left-hand_side_of_an_assignment

So why drop `return false`?

Reply:
@fregante this is the only place in the code that I found a simple return false that follows an if.

In our words why here do we do return false and in all other features that get swallowed by the first `if` we do not.

Reply:
> all other features that get swallowed by the first `if` we do not.

That's a problem with those features, rather than this one. It's not a huge problem given it only changes the debug logger, but in this case I think it also affects the `shortcuts` field of `features.add` (and if it doesn't, it should)

I don't think this feature needs any change